packages/app/src/components/choropleth/tooltips/tooltipContent.tsx

```diff
@@ -29,11 +34,11 @@ export function TooltipContent(props: IProps) {
   );
 }
 
-const StyledTooltipContent = styled.div((x) =>
+const StyledTooltipContent = styled.div((x: any) =>
```

Comment:
You should define the props/interface than you don't need any

```suggestion
const StyledTooltipContent = styled.div<{onClick?: ()=>void}>((x) =>
```

Reply:
Or even better do not pass the click handler to this component, as it doesn't care. Instead pass in a boolean or something `hasClickHandler` that you derive from it

Reply:
I'm wondering why the initial change from `styled.div((x) =>` to `styled.div((x; any) =>` was necessary at all, I'd think that `x` would infer the type coming from styled components (including the `onClick` type).